cloud-functions/functions/src/index.ts

```diff
@@ -440,3 +440,50 @@ export const initiatePasswordRecovery = functions.https.onCall((body) => {
       });
   });
 });
+
+export const getVerificationCode = functions.https.onCall(async () => {
+  const axios = require('axios');
+  const axiosCookieJarSupport = require('axios-cookiejar-support').default;
+  const tough = require('tough-cookie');
+  const qs = require('querystring');
+  axiosCookieJarSupport(axios);
+
+  let config = functions.config().verif_server;
+  let url = config.url;
+
+  const cookieJar = new tough.CookieJar();
+  let instance = await axios.create({
+    jar: cookieJar,
+    withCredentials: true,
+  });
+
+  try {
+    let response = await instance.post(
+      'https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key=' + config.key,
```

Comment:
Out of curiosity, where did you find this url? Do others do this non-firebase-client firebase auth login elsewhere (I imagine so)?

Reply:
I got this from logging in to the Google Verif Server UI and looking at the network requests it made in the browser dev tools.

Reply:
Nice

Reply:
FWIW [here](https://developers.google.com/identity/toolkit/web/reference/relyingparty/verifyPassword) is the official documentation for this API endpoint